lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -515,7 +515,7 @@ mlir::linalgx::packVNNIMatmulOp(RewriterBase &rewriter,
   return replacementOp;
 }
 
-FailureOr<vnni::BRGemmOp>
+FailureOr<tpp::BrgemmOp>
 mlir::linalgx::packVNNIBRGemmOp(RewriterBase &rewriter,
```

Comment:
We don't seem to have a `packBRGemmOp`, is that expected?

Reply:
We never packed BRGEMM

Reply:
Why do we pack VNNI brgemm and not standard brgemm?

We don't want to have VNNI specific functions at this level, because the end result is that they're very similar but different. We need a way that doesn't make them different, relying on composition (template, pimpl, etc).

Reply:
No idea, why it is like this.

Reply:
@KavithaTipturMadhu can you shed light on this?